app/views/advanced_search_finder/_results.mustache

```diff
@@ -11,14 +11,13 @@
     for visual consistency, this might prove to be a maintenance headache in which case it would be better
     to port the styles to the advanced-search sass file. }}
     <li class="gem-c-document-list__item">
-      <h3 class="gem-c-document-list__item-title">
-        <a href='{{document.link}}'
-           data-track-category='navFinderLinkClicked'
-           data-track-action='{{finder_name}}.{{document_index}}'
-           data-track-label='{{document.link}}'
-           data-track-options='{"dimension28":"{{page_count}}","dimension29":"{{document.title}}"}'
-        >{{document.title}}</a>
-      </h3>
+      <a href='{{document.link}}'
+        class="gem-c-document-list__item-title document__link"
```

Comment:
This is interesting... the class of `document__link` here doesn't appear to have any effect (see https://finder-frontend-pr-889.herokuapp.com/search/advanced?group=services&topic=%2Feducation).

I think now we're using a specific class for the link, we could take the opportunity to un-nest it from beneath the `#finder-frontend` selector, which could mean it could apply here and we could remove the `gem-c-document-list__item-title` class.

Sorry to continually muddy what should have been a simple PR!